GitUI/CommandsDialogs/SettingsDialog/EditorHelper.cs

```diff
@@ -1,61 +1,49 @@
 using GitCommands;
 
-namespace GitUI.CommandsDialogs.SettingsDialog.Pages
+namespace GitUI.CommandsDialogs.SettingsDialog.Pages;
+
+public static class EditorHelper
 {
-    public static class EditorHelper
-    {
-        public static string FileEditorCommand
-            => $"\"{AppSettings.GetGitExtensionsFullPath()}\" fileeditor";
+    public static string FileEditorCommand
+        => $"\"{AppSettings.GetGitExtensionsFullPath()}\" fileeditor";
 
-        public static string[] GetEditors()
+    public static string[] GetEditors()
+    {
+        return new[]
         {
-            return new[]
-            {
-                FileEditorCommand,
-                "vi",
-                "notepad",
-                GetNotepadPP(),
-                GetSublimeText3(),
-                GetVsCode(),
-                GetAtom()
-            };
-        }
+            FileEditorCommand,
+            "vi",
+            "notepad",
+            GetNotepadPlusPlus(),
+            GetSublimeText(),
+            GetVsCode(),
+        };
+    }
 
-        private static string GetNotepadPP()
-        {
-            return GetEditorCommandLine("Notepad++", "notepad++.exe", " -multiInst -nosession", "notepad++");
-        }
+    private static string GetNotepadPlusPlus()
+        => GetEditorCommandLine("notepad++.exe", "-multiInst -nosession", "notepad++");
 
-        private static string GetVsCode()
-        {
-            return GetEditorCommandLine("Visual Studio Code", "code.exe", " --new-window --wait", "Microsoft VS Code");
-        }
+    private static string GetVsCode()
+        => GetEditorCommandLine("code.exe", "--new-window --wait", "Microsoft VS Code");
 
-        private static string GetAtom()
-        {
-            return GetEditorCommandLine("Atom", "atom.exe", " --wait", "atom");
-        }
+    // http://stackoverflow.com/questions/8951275/git-config-core-editor-how-to-make-sublime-text-the-default-editor-for-git-on
+    private static string GetSublimeText()
+        => GetEditorCommandLine("sublime_text.exe", "--wait --multiinstance", "Sublime Text", "Sublime Text 3");
```

Comment:
```suggestion
        => GetEditorCommandLine("sublime_text.exe", "--wait --multiinstance", "Sublime Text", "Sublime Text");
```

Reply:
This is a special call syntax the last arguments form a `string[]`.

Reply:
The path can be removed, no v3 release since 2019